src/patterns/navigate-a-service/index.md

```diff
@@ -0,0 +1,214 @@
+{% endcall %}
+
+#### External links
+
+Links that go outside of your service should usually:
+
+- be grouped together, and shown after links within your service
+- open in a new window – warn the user of this if possible
+
+Consider showing external links in the page body instead.
+
+### GOV.UK One Login
+
+GOV.UK One Login [publishes its own header component](https://www.sign-in.service.gov.uk/documentation/design-recommendations/let-users-navigate-sign-out) for services to give users an easy, consistent route to their GOV.UK One Login and a way to sign out.
+
+## Adding other header and navigation elements
+
+To help maintain consistency across services on GOV.UK, we’ve outlined some guidance on how to use the space in your header, and where to add other components and elements.
+
+As a rule, the ordering of header and navigation elements should start with most general (GOV.UK-wide) elements at the top, with the more specific (service-level) elements further down.
+
+### Start with GOV.UK-wide, general elements
+
+In the GOV.UK header, only add GOV.UK-wide elements.
+
+### Next, show any important elements that affect your entire service
+
+You can add space between the GOV.UK header and Service navigation to insert important elements that apply to your entire service.
+
+Elements you might add here include:
+
+- a [Phase banner](/components/phase-banner/)
+- an ‘organisation switcher’, or a similar tool that helps users use your service across a set of things
+
+See an example of an [organisation switcher](https://design-patterns.service.justice.gov.uk/components/organisation-switcher/) component in the Ministry of Justice Design System.
+
+We’d like to learn more about elements you might place between the GOV.UK header and Service navigation. Tell us in our discussion space [LINK].
+
+### Then, add any other service-level elements
+
+Most other service-level elements can be added within the Service navigation, immediately after the service name.
+
+The Service navigation component [includes ‘slots’ to support this](/components/service-navigation/#use-slots-to-add-custom-elements). You’ll need to decide on the most appropriate layout and positioning and provide your own styles.
+
+Elements you might add to Service navigation include:
+
+- language selectors, if they work across your entire service
+- search inputs, if they only search within your service
+
+To help users understand what the search input will cover, include ‘Search [your service]’ as placeholder text within the search input.
+
+### Finally, show page-specific elements
+
+Show any elements that only affect the current page after the Service header navigation.
+
+Place any [Breadcrumbs](/components/breadcrumbs/) just before the `<main>` element. Placing them here means that the ‘Skip to main content’ link allows the user to skip all navigation links, including breadcrumbs.
```

Comment:
Something weird is happing with the formatting, '<main>' code syntax is wrong?

Reply:
I think this one is fixed now